src/utils/log_metrics.py

```diff
@@ -122,3 +122,21 @@ def std_var_events_in_log(log: EventLog) -> int:
     :return 3
     """
     return statistics.stdev([len(trace) for trace in log])
+
+
+def trace_ids(log: EventLog) -> list:
```

Comment:
Please consider writing it as follows:

def trace_ids(log: EventLog) -> list:
    return [trace.attributes[NAME_CLASSIFIER] for trace in log]

It is faster and less congnitively complex, you can read more about the latter in (https://www.sonarsource.com/docs/CognitiveComplexity.pdf)